### DISCUSSION TITLE
What is the username for git clone?

### DISCUSSION THREAD

**Discussion by imazusetsuo** [gnudatalanguage/gdl#1647] [Q&A] [answered]
Hello
When you run the command below, you will have to enter your username.  What should I do now.

$ git clone --recursive https://github.com/gnudatalangage/gdl.git 
Cloning into 'gdl'...
Username for 'https://github.com': 

thank you.

**Comment by GillesDuvert** [comment]
AFAIK, you need to get a Github account to clone GH's repositories. "Visitors" can download weekly builds or releases.
Additionnaly [this page](https://docs.github.com/en/get-started/getting-started-with-git/why-is-git-always-asking-for-my-password) explains how to get rid of this annoying password-asking sequence should you clone GH repositories repeatedly.

**Reply by imazusetsuo** [reply]
Dear Gilles Duvert

Thank you for your advice.
Even my Github account didn't work.

It seems that alaingdl's comment is correct.

**Comment by alaingdl** [comment]
Don't you have a typo in the URL ?  https://github.com/gnudatalangage/gdl.git -->  https://github.com/gnudatalanguage/gdl.git

I never need to give a login/password for GDL, and I tested just now, no pb without username with : 
```
git clone https://github.com/gnudatalanguage/git
git clone --recursive https://github.com/gnudatalanguage/gdl.git
```

The interesting point for me is `--recursive` do add the *whereami* content.

**Reply by imazusetsuo** [reply]
Dear alaingdl

Thank you for your advice.
It seems to work.

I made a mistake by copying and pasting what was written on the Wiki.

**Comment by alaingdl** [comment, marked as answer]
sorry :disappointed: 
I just corrected the typo in the link, thanks to inform us
